### PR TITLE
Refactor/styles

### PR DESCRIPTION
**Pull request by Karlfunhouse**
Started styling css but then had to refactor some js to get it to not print 2 winner cards.  Figured out we had to check both guesses before running the gameWinner function.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -137,10 +139,13 @@
 
   function checkGuesses() {
     numberOfGuesses += 2;
-    guessOneDisplay.innerText = challengerGuesses[0].value;
-    guessTwoDisplay.innerText = challengerGuesses[1].value;
-    checkGuess1();
-    checkGuess2();
+    var guess1 = checkGuess1();
+    var guess2 = checkGuess2();
+    if (guess1 == randomNumber) {
```
Comment (KmBarnett) [inline main.js:144]: We did this as a fix to a winner reseting the number before we checked the next guess

File: styles.css [hunk 2]
```diff
@@ -8,13 +8,13 @@ body,
...
   width: 100%;
-  height: 100%
+  height: 100%;
 }
 
 main {
   display: flex;
   flex-direction: column;
-  height: 100%
+  /* height: 100%; */
```
Comment (KmBarnett) [inline styles.css:17]: Lets delete  this comment out we are already deciding the width and height of html, body, main, and container set to the same value

File: styles.css [hunk 3]
```diff
@@ -110,12 +110,13 @@ h1 {
 .left-side {
   background-color: #f7f7f7;
   width: 50%;
+  height: fit-content;
 }
 
 .right-side {
   background-color: #d0d2d3;
-  height: 100vh;
```
Comment (KmBarnett) [inline styles.css:117]: looks good

File: styles.css (hunk 2 above)
Comment (Karlfunhouse) [reply styles.css:17]: cool all set

**Pull request merged**